data/json/items/migration.json

```diff
@@ -20,6 +20,12 @@
     "replace": "battery_car"
   },
   {
+    "id": "gold_small",
+    "type": "MIGRATION",
+    "replace": "gold",
+    "charges": 1
```

Comment:
It's actually even worse, it will convert *any* stack of silver / gold to 1 charges. This means a player that has accumulated "gold (10000)" (has disassembled all gold bars) ends up with a single charge of it.

I'm not generally opposed to it (your reasoning and having access to the debug menu), but it should be mentioned in the PR.